utils/uploads_used.py

```diff
@@ -80,3 +84,43 @@ def query_uploads_used(plan_service, limit, owner):
         )
 
     return queryset[:limit].count()
+
+
+def query_monthly_coverage_measurements(plan_service: PlanService) -> int:
```

Comment:
How will this be used? How come it only does private repo?

Reply:
This is currently not in use but will replace the `query_uploads_used` fn. It's mimicking the logic there, excluding the `uploaded` filter (there's 2 types so far, uploaded and carriedforward; this new user_measurements table will only insert uploaded entries, so no need for it anymore) and the `...timedelta(days=60)` as that was a stab at performance, which this whole approach will take care of now